### PR TITLE
Symfony

### PR DESCRIPTION
**Pull request by fideloper**
Mention Symfony in the Laravel/PHP docs, since it's just a few tweaks from Laravel's setup.

See https://dejager.sh/articles/deploying-symfony-on-fly

### CODE REVIEW

File: app-guides/mysql-on-fly.html.md [hunk 1]
```diff
@@ -151,7 +152,7 @@ flyctl proxy 13306:3306 -a my-mysql
 Then connect to your MySQL server at `localhost:3306` and the username and password credentials from above:
 
 ```cmd
-mysql -h localhost -P 3306 -u hotdogs_are_tacos -ppassword cube_theory
+mysql -h localhost -P 3306 -u hotdogs_are_tacos -ppassword some_db
```
Comment (KTanAug21) [inline app-guides/mysql-on-fly.html.md:155]: change hotdocs_are_tacos to non_root_user as well here, as changed in line 70 above

Comment (KTanAug21) [reply app-guides/mysql-on-fly.html.md:155]: also! just a quick point out the title of the current branch is for Symfony changes

**Comment by florentdestremau** [comment]
Where the docs on Symfony tested before being linked ? The article is almost blank and tons of additional work is needed to run any basic app.

**Comment by florentdestremau** [comment]
Yes I know but the article itself is far from sufficient. I tried it out yesterday for a simple side project and it took me 1 hour to get to absolutely nothing. The build does not work out of the box with the instructions:

- APP_ENV is not set in docker so it's building the image with dev mode
- it's still trying to read a `.env` file
- it's trying to find a node js project where there isn't

The list goes on. This is most definitely not enough to have a small project running. I appreciate the fact that you linked a community effort in your docs but to me it's misleading as you haven't tried it to make sur it is enough. I wont have the time this month to go further into it but for a side project of mine I'll try to push as far as I can go to make a real tutorial.

**Comment by fideloper** [comment]
Gotcha, thanks for the pointers. I'll try it out soon to see what else I can dig up on that.

**Pull request merged**